pkiintergration-fe/src/services/create-certificate.service.ts

```diff
@@ -0,0 +1,21 @@
+import { Injectable } from '@angular/core';
+import { FetchClient } from '@c8y/client';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class CreateCertificateService {
+  private options = {
+    method: 'GET',
+    headers: { 'Content-Type': 'application/json' }
+  };
+
+  constructor(private fetchClient: FetchClient) {}
+  async getCertificate() {
```

Comment:
todo: function return type not defined

make use of typescript and set a retrun type definiton, even if the return is void